src/anemoi/models/models/encoder_processor_decoder.py

```diff
@@ -69,6 +70,31 @@ def __init__(
 
         self.num_channels = config.model.num_channels
 
+        # If self.layer_kernels entry is missing from the config, use torch.nn by default
+        try:
+            self.layer_kernels = config.model.layer_kernels
+        except AttributeError:
+            LOGGER.info("No entry found for config.model.layer_kernels. Defaulting to torch.nn implementations")
```

Comment:
better to error out gracefully, perhaps?

Reply:
good idea, done